test/controllers/address/PostcodeLookupControllerSpec.scala

```diff
@@ -419,58 +357,44 @@ class PostcodeLookupControllerSpec extends BaseSpec {
       def addressLookupResponse: RecordSet = RecordSet(List())
 
       when(mockAddressLookupConnector.lookup(any(), any())(any(), any())) thenReturn
-        EitherT[Future, UpstreamErrorResponse, RecordSet](
-          Future.successful(
-            Right(addressLookupResponse)
-          )
-        )
+        EitherT[Future, UpstreamErrorResponse, RecordSet](Future.successful(Right(addressLookupResponse)))
       when(mockJourneyCacheRepository.get(any[HeaderCarrier])).thenReturn(
         Future.successful(
           UserAnswers
             .empty("id")
             .setOrException(AddressFinderPage(PostalAddrType), AddressFinderDto("AA1 1AA", None))
-            .setOrException(AddressLookupServiceDownPage, false)
         )
       )
       when(mockJourneyCacheRepository.set(any[UserAnswers])).thenReturn(Future.successful((): Unit))
       when(mockCitizenDetailsService.personDetails(any())(any(), any(), any())).thenReturn(
-        EitherT[Future, UpstreamErrorResponse, Option[PersonDetails]](
-          Future.successful(Right(Some(personDetails)))
-        )
+        EitherT[Future, UpstreamErrorResponse, Option[PersonDetails]](Future.successful(Right(Some(personDetails))))
       )
 
       def currentRequest[A]: Request[A] =
         FakeRequest("POST", "/test")
           .withFormUrlEncodedBody("postcode" -> "AA1 1AA")
           .asInstanceOf[Request[A]]
 
-      val result: Future[Result] = controller.onSubmit(PostalAddrType, None)(currentRequest)
+      val result: Future[Result] = controller.onSubmit(PostalAddrType, Some(true))(currentRequest)
```

Comment:
This change is suspicious. can you explain?
Given https://github.com/hmrc/pertax-frontend/blob/a8d6ea87cdc89bc7154cdfc01f676ee9b8f10c90/app/controllers/address/PostcodeLookupController.scala#L130 None default to false.